packages/terra-worklist-data-grid/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 * Changed
   * Removed roving tabIndex and resolved focus management issues within cells.
+  * Updated the functionality for selecting a range of rows.
```

Comment:
Are there API changes for row selection?

Reply:
Yes, based on discussion from post scrum. We made the update.